packages/app-content-pages/pages/about.js

```diff
@@ -0,0 +1,16 @@
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+export { default } from '../src/screens/About/About.js'
+
+export async function getStaticProps({ locale }) {
+  try {
+    return {
+      props: {
+        ...(await serverSideTranslations(locale, ['components']))
+      }
```

Comment:
Once the stats ("classifications so far" and maybe "registered volunteers") are added will we want to revalidate similar to teams or publications? Not needed for this PR, other than maybe to add a comment if we will.

Reply:
Good point! Will consider it in future PRs